run_dir/design/flowcell_trend_plot.html

```diff
@@ -15,45 +15,53 @@ <h1>Flowcells Trend Plot</h1>
 <div id="main_plot" style="height: 700px;">
 </div>
 <div id="control_container" style="margin-top:40px;">
-    <div class="row">
-        <div id="logs_date" class="col-md-4">
-            <form class="form-inline" id="logs_date_form">
-                Select flowcells from 
+    <div class="row p-3">
+        <div id="logs_date" class="col-md-3 d-flex justify-content-center border border-secondary p-3 rounded-3">
+          <form class="form-inline" id="logs_date_form">
+            <div class="row">
+              <div class="col-10">
+                Select flowcells from
                 <div class='input-group input-group-sm date' id='datepick1'>
-                    <input id="inp_date_1" type='text' class="form-control" /><span class="input-group-addon"><span class="glyphicon glyphicon-calendar"></span></span>
+                    <input id="inp_date_1" type='text' class="form-control" /><span class="input-group-addon"><span class="input-group-text fa fa-calendar-alt"></span></span>
                 </div>
-            to
-            <div class='input-group input-group-sm date' id='datepick2'>
-                <input id="inp_date_2" type='text' class="form-control"/><span class="input-group-addon"><span class="glyphicon glyphicon-calendar"></span></span> 
+                to
+               <div class='input-group input-group-sm date' id='datepick2'>
+                <input id="inp_date_2" type='text' class="form-control"/><span class="input-group-addon"><span class="input-group-text fa fa-calendar-alt"></span></span>
+               </div>
+              </div>
+             <div class="col-2 d-flex justify-content-center align-self-center">
+              <button id="submit_interval" class="btn btn-primary btn-sm mt-3">Search</button>
+             </div>
             </div>
-            <button id="submit_interval" class="btn btn-primary btn-sm">Search</button>
-            </form>
+          </form>
         </div>
-        <div class="col-md-4">
+        <div class="col-md-5 d-flex justify-content-center">
+          <div class="d-flex justify-content-center border border-secondary p-3 rounded-3" style="box-sizing: border-box; -moz-box-sizing: border-box; width: 50%">
```

Comment:
Don't like to have the 'style' attribute at all. The width should be possible with the `w-50` class right?

Reply:
But feel free to keep it, its in so many other places on this page, so might not make much of a difference. It is entirely possible that it's I that need to change as well.

Reply:
Got rid of some of it.